include/multipass/exceptions/exitless_sshprocess_exception.h

```diff
@@ -40,6 +42,7 @@ class ExitlessSSHProcessException : public std::runtime_error
 
 private:
     const std::string cmd;
+    const std::string cause;
```

Comment:
Not sure why we need to have `cause` as a member of the class since the `runtime_error` ctor is called during the `ExitlessSSHProcessException` ctor and `cause` is never used afterwards.

Along with that, but separate from this PR, why is `cmd` used as well?  I see the `command()` accessor method that uses `cmd`, but I don't see it used anywhere.  Not a blocker for this PR but just curious...

Reply:
Both are superfluous indeed. When I added the exception I tried to stick with what was done in `mp::StartException`, which also keeps the instance name in an attribute which I don't think is ever used. This time I just repeated the same thing, but I notice now I forgot to add the corresponding accessor method...

Reply:
So, I can either remove both attributes, arguably breaching this PR's scope, or add an accessor method, keeping attributes for no real reason. If you agree I will remove them.

Reply:
Right, the `name()` accessor is used in the daemon code.  I think it's safe to remove the private members and the accessor method.  If we ever need them in the future, it's trivial to add:)

Reply:
Thanks!